istio.io/configmap-nginx.yaml

```diff
@@ -84,6 +84,14 @@ data:
           return 200 "ok\n";
         }
 
+        # Proxy to past versioned instances of the site.
+        location ~ ^/v-[0-9]+\.[0-9]+.*$ {
+          proxy_set_header X-Real-IP $remote_addr;
+          proxy_set_header X-Forwarded-For $remote_addr;
+          proxy_cache      off;
+          proxy_pass       https://istiodocs-v01.firebaseapp.com:443;
```

Comment:
This should be handled via `upstream` block in order to enable keepalive connections (see example for `upstream github { ... }`).